draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2151,19 +2152,19 @@ uCDN-A                         dCDN-B                         dCDN-C
         "type": "hls"
       }
     }
-   ],
+  ],
   "objects": {
-     "href": "https://dcdn.com/triggers/2a544cd9/objects",
-     "type": "json"
-  }
+    "href": "https://dcdn.com/triggers/2a544cd9/objects",
```

Comment:
This looks wrong.  According to the documentation, this is an array of ObjectList objects, and this doesn't look like an array.  Shouldn't this be surrounded by [ ]?